src/sentry/static/sentry/app/views/dashboards/exploreWidget.jsx

```diff
@@ -128,8 +128,8 @@ class ExploreWidget extends React.Component {
         target={!flags.discover2 ? '_blank' : ''}
         title={
           flags.discover2
-            ? t('Explore data in the new Discover')
-            : t('You do not have access to the new Discover. Click to learn more.')
+            ? t('Explore data in the Discover')
+            : t('You do not have access to the Discover. Click to learn more.')
```

Comment:
The wording here feels a little clunky. Should it not be `Explore data in Discover` or `You do not have access to Discover`?